src/System.Security.Cryptography.OpenSsl/tests/EcDsaOpenSslImportExportTests.cs

```diff
@@ -0,0 +1,339 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+using Test.Cryptography;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace System.Security.Cryptography.EcDsa.Tests
+{
+    public static class EcDsaImportExportTests
+    {
+        public class CurveDef
+        {
+            internal CurveDef() { }
+            internal Oid Id;
+            internal int KeyLength;
+            internal ECCurveType CurveType;
+            internal bool IncludePrivate;
+        }
+
+        [Fact]
+        public static void ExportNamedCurve()
+        {
+            ECDsaOpenSsl ec = null;
+            ec = new ECDsaOpenSsl(ECNamedCurve.FromOidValue("1.2.840.10045.3.1.7")); //nistP256
+            ECParameters param = ec.ExportNamedCurveParameters(false);
+            Assert.Equal(256, ec.KeySize);
```

Comment:
And that it's actually a named curve param object; and that the OID is right; and that all the fields have the right lengths... (since this is new surface we get to decide between "we'll guarantee zero-padded diminished values on export, and require them on import" or "we'll zero-pad them when sending them to Windows, since Windows uses one `cb` for lots of fields")